src/app/shared/components/product/product-tile/product-tile.component.html

```diff
@@ -34,20 +34,20 @@
 
```

Comment:
this one above could become a switch for `displayVariationCount`

Reply:
?